internal/cmd/version/version_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build integration
+// +build integration
```

Comment:
for now we should only go with `// +build` annotations, since `//go:build` was introduced in Go 1.17

Reply:
This one was added by the tooling. I don't think it hurts 1.16, right?